### ISSUE TITLE
mailto: schema URL Parameters

### ISSUE THREAD

**Issue by xen** [maizzle/framework#324]
- Maizzle Version: ^2.3.0
- Node.js Version: v14.13.0

Hello! 

First, thank you very much for your amazing product. 

I'm trying to use full features of the project and added support for URL parameters. This is my configuration file:

```js

module.exports = {
  // ...skip...
  urlParameters: {
    _options: {
      tags: ['a'], 
      qs: {},
    },
    utm_source: 'mail',
    utm_campaign: 'transaction',
    utm_medium: 'email',
  }
};

```

I want to add to my template mailto: link using `<a>` tag. For example this is code I have:

```
<a
  href="mailto:support@okumy.com?subject=Ask support"
  class="hover:text-brand-700 hover:underline"
>ask support</a>
```

It is rendered to this code:

```html
<a href="mailto:support@okumy.com?utm_campaign=transaction&utm_medium=email&utm_source=mail" class="hover-text-brand-700 hover-underline" style="color: #0047c3; text-decoration: none; display: inline-block">ask support</a>
```

`?subject=` part of the mailto link is gone and link contains utm_* parts which is not necessary here. 

I thought that it is possible to protect this piece by using `<raw></raw>` code, but it doesn't work.

**What I want and expect**: mailto links doesn't change and no any additional parameters will be added using this scheme.

**Comment by cossssmin** [comment]
Yes, it's because `mailto:....` is a valid URL. We use [is-url-superb](https://github.com/sindresorhus/is-url-superb) in the [posthtml-url-parameters](https://github.com/posthtml/posthtml-url-parameters) plugin that adds parameters to URLs in `href` attributes, and it matches as a valid URL so the PostHTML plugin parses it...

The solution here would probably be an option for `posthtml-url-parameters`, to define which protocol(s) it should skip over.

`<raw>` tag is not meant for this, it only prevents parsing [posthtml-expressions](https://github.com/posthtml/posthtml-expressions) functionality, like expressions in `{{ }}` brackets.

**Comment by xen** [comment]
I'm not very familiar with JS ecosystem. Is there any quick workaround you can recommend? I looked into posthtml-url-parameters it doesn't have any configuration options to exclude protocols.

**Comment by cossssmin** [comment]
Sorry, I meant `posthtml-url-parameters` could use an option like that, it’s not added yet. 

Since the plugin only parses `href` attributes, you could use `data-mailto-href` for your `mailto` links and then use `replaceStrings` in Maizzle to rewrite them back to a `href` attribute:

https://maizzle.com/docs/code-cleanup/#replacestrings

**Comment by xen** [comment]
Thank you, this helped.

I created replace string that looks like entity and replace it as needed.
